data/questions/disquetusais.js

```diff
@@ -212,10 +213,11 @@ réagir calmement aux désaccords et provocations. »
     },
     correctAnswers: ['a'],
     explanation: "Dans la mesure où il a fait échouer la passe des attaquants, le défenseur récupère le disque (même s’il est sorti du terrain).",
-    rules: undefined,
+    rules: ["11.2."],
     source: "Disque Tu Sais - Pablo",
     level: Levels.MIDDLE,
   },
+    /*  no rules found in rules or annotations
```

Comment:
C'est dans les annotations (A5.4.1)

https://rules.wfdf.org/documents/wfdf-rules-of-ultimate-2021-2024-appendix-v2-0/download

Je pense qu'on peut laisser la question et indiquer dans l'explication comment trouver la réponse précise. Le jour où les annotations seront traduites on pourra mettre la bonne référence

Reply:
ok